.github/workflows/integration.yml

```diff
@@ -0,0 +1,100 @@
+name: bootc integration test
```

Comment:
Biggest question: Will we *also* continue to run the tests via packit+testing farm? Probably not, right?

But still though, we should bear in mind that we still need to have good bridges/integration with at least tmt/testing-farm since other groups will use that.